### PR TITLE
Portfolio feature

### PR DESCRIPTION
**Pull request by ernestkorir**
Display Content through Javascript to the template in an HTML file

Thanks for reviewing my PR 
Hoping for the best

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,7 +1,96 @@
...
+        liveLink:'https://ernestkorir.github.io/my-portfolio/', 
+        sourceLink: 'https://github.com/ernestkorir/my-portfolio.git',
+        year: 2021
+    }
+    
+]
+
+const loadProjects = function() {
+    let projectsDiv = document.querySelector('section#projects');
+    let cardTemplate = projectsDiv.querySelector('#project-template');
```
Comment (ShoiraTa) [inline script.js:51]: - [ ] When I click on project I get the popup with completely different topic and images. The same popup shows up on every click,. 

https://user-images.githubusercontent.com/77038610/195678879-662c4df5-506c-4e20-963b-0bb0ba771117.mp4

File: style.css [hunk 2]
```diff
@@ -626,7 +634,7 @@ body {
   .card-work-1 {
     margin: 142px;
     flex-direction: row;
-    height: 496px;
+    height: 50%;
     left: 0;
     top: 0;
     border-radius: 24px;
```
Comment (ShoiraTa) [inline style.css:669]: - [ ] The popup should follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins)

Your desktop design   | Figma desktop design
------------- | -------------
 Screenshot 2022-10-13 at 21 40 50 |  Screenshot 2022-10-13 at 21 41 10

Your mobile design  | Figma mobile design
------------- | -------------
Screenshot 2022-10-13 at 21 42 46 | Screenshot 2022-10-13 at 21 43 32

Please make sure that you have the same layout, eg., text placement margins paddings etc.

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @ernestkorir ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- Professional pull request! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username [@shoirata].


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -817,9 +833,38 @@ body {
...
+  }
+
+  #error {
+    display: none;
+  }
+
+  #card-work-1 {
+    display: block;
+    width: 100%;
+  }
```
Comment (levy002) [inline style.css:867]: - [ ] Kindly make sure to meet the provided Figma as requested by the previous reviewer. :wink: 

   - ### Your designs
   image

    image


   - ### Figma designs
   image

   image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @ernestkorir   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅
## Hi @ernestkorir 👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**

 

## Highlights:
* Well written PR description ✔️ 
* All tests passed ✔️ 

## Optional suggestions:
- [ ] It would be better if the popup window wasn't a bit transparent, it's a bit hard to look at the content this way 👍 

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**